ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -114,6 +114,12 @@
     reason="Test runs ONLY on IPI deployed cluster"
 )
 
+skipif_aws_i3 = pytest.mark.skipif(
+    config.ENV_DATA['platform'].lower() == 'aws'
+    and config.DEPLOYMENT.get('local_storage') is True,
+    reason="Tests are not running on aws i3 instances"
```

Comment:
can this skip reason be changed to "Tests will not run on AWS i3"?
WDYT?

Reply:
changed